src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -185,6 +194,28 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
         return new Promise((resolve, reject) => api.getVersionsAPI(false).getVersions(fileId, resolve, reject));
     };
 
+    fetchCurrentVersion = ([fileResponse, versionsResponse]): Promise<[BoxItem, FileVersions, FileVersions]> => {
```

Comment:
Super minor, but all the `fetch` methods in here are alphabetized at the moment. Maybe `fetchVersionCurrent`? 😄